src/no-port-picked/no-port-picked-dialog.ts

```diff
@@ -29,27 +50,56 @@ class EwtNoPortPickedDialog extends LitElement {
             Most devices have a tiny light when it is powered on. If yours has
             one, make sure it is on.
           </li>
+          <li>
+            Make sure that the USB cable you use can be used for data and is not
+            a power-only cable.
+          </li>
           <li>
             Make sure you have the right drivers installed. Below are the
             drivers for common chips used in ESP devices:
             <ul>
               <li>
-                CP2102 (square chip):
+                CP2102 drivers:
                 <a
                   href="https://www.silabs.com/products/development-tools/software/usb-to-uart-bridge-vcp-drivers"
                   target="_blank"
                   rel="noopener"
-                  >driver</a
+                  >Windows & Mac</a
                 >
               </li>
               <li>
-                CH341:
+                CH9102 drivers:
```

Comment:
I'd mention CH342, CH343 as well, as I think they are around too

```suggestion
                CH342, CH343, CH9102 drivers:
```